### ISSUE TITLE
linerange issues

### ISSUE THREAD

**Issue by mihaicristiantanase** [nicklockwood/SwiftFormat#688]
I'm using `--linerange` (swiftformat 0.45.3) option and I've found 2 issues using the following file:

```swift
// indentation with 2 spaces
func run() {
  if data[idx] != nil {
        print(            "has data") /* this line needs indentation fix */
  }
}
```

**First issue** - linerange formatting needs more context than necessary:

Actual:
Running
```bash
swiftformat --indent 2 --swiftversion 5 --quiet --linerange 3,4
```
removes the extra spaces after the open parenthesis of `print` function.

The actual fix is to run:
```bash
swiftformat --indent 2 --swiftversion 5 --quiet --linerange 2,4
```

Expected:
Running
```bash
swiftformat --indent 2 --swiftversion 5 --quiet --linerange 3,4
```
should be sufficient for fixing the line.

**Second issue** - error formatting

Actual:
```bash
swiftformat --indent 2 --swiftversion 5 --quiet --linerange 4,5
```
shows the following error: `Formatting /usr/local/bin/bash: line 1:  3534 Illegal instruction: 4`

Expected:
No error because the input is a valid Swift file.

**Comment by nicklockwood** [comment]
Thanks for reporting, I'll look into it.

**Comment by nicklockwood** [comment]
@mihaicristiantanase I believe all of these issues are now fixed in 0.45.4. Let me know if you find any other problems.

**Comment by mihaicristiantanase** [comment]
I can confirm that 0.45.4 fixes the issues.
Thank you!
